landing_pages/mark/mark.js

```diff
@@ -31,8 +31,8 @@ function getCode(){
 function leaveComment(){
 	current_form = "leave_comment"
 	data = {
-		"email": document.getElementById('email').value,
-		"message": document.getElementById('comment').value,
+		"email": document.getElementById('email').getElementsByClassName('visible')[0],
```

Comment:
Do getElementsByClassName first -> get elements array and then do get element by id on each element

or simply instead of attaching/detaching a class visible attach elementid_visible, that way you can do get elements by class and get single unique element in array